src/sdk.ts

```diff
@@ -368,7 +370,8 @@ export class DefaultRelayingServices implements RelayingServices {
                         tokenAmount: await this.web3Instance.utils.toWei(
                             tokenAmount.toString()
                         ),
-                        onlyPreferredRelays: true
+                        onlyPreferredRelays: true,
+                        ...transactionDetails
```

Comment:
if the parameter is omitted when calling `relayTransaction`, will this line work ok?

Reply:
it's clear, thanks!